modules/cms/helpers/File.php

```diff
@@ -68,4 +71,24 @@ public static function validatePath($filePath, $maxNesting = 2)
 
         return true;
     }
+
+    /**
+     * Validates a CMS object path is inside the application's base directory.
+     * @param string $filePath Specifies a path to validate
+     * @return boolean Returns true if the file path is local. Otherwise returns false.
+     */
+    public static function validateIsLocalFile($filePath)
+    {
+        $restrictBaseDir = Config::get('cms.restrictBaseDir', true);
+
+        if ($restrictBaseDir && !app(Filesystem::class)->isLocalPath($filePath)) {
```

Comment:
@tobias-kuendig You can use the `File` facade instead of directly interacting with the application provider.
```suggestion
        if ($restrictBaseDir && File::isLocalPath($filePath)) {
```

Reply:
Good point!